src/actions/PageActions.js

```diff
@@ -9,6 +9,8 @@ export function setPage(page) {
 
 export function setEvent(event) {
   if (!event.id) {
+    let newurl = window.location.protocol + "//" + window.location.host + window.location.pathname;
+    window.history.pushState({ path: newurl }, '', newurl);
```

Comment:
if `setEvent()` was called with no event (or specifically an object with no `id`), send the user to homepage and set the URL to have no query param (else line 20 sets the URL to have `'?event=' + event.id;`)

It might be hard to read that it is an else, if you don't realize the if will return and not run the rest. I can make it an else for readability.

Reply:
Also I'm not sure if its confusing to be able to set the page by calling `setEvent`. Let me know if you think that doesn't make sense.

Reply:
> Also I'm not sure if its confusing to be able to set the page by calling `setEvent`. Let me know if you think that doesn't make sense.

Yeah perhaps change the variable names for our "organization events" to something other than Event

Reply:
These actually are "organization events"
`setEvent` sets the "current" event of the app ie going to app.ubcbiztech.com/event?=cfn2 will call `setEvent` with the event id `cfn`